test/collins/controllers/AssetApiHelper.scala

```diff
@@ -54,9 +54,9 @@ trait AssetApiHelper extends ResponseMatchHelpers with JsonMatchers with Control
     result
   }
 
-  def updateHwInfo() = {
+  def updateHwInfo(lshwXml: String = "lshw-basic.xml") = {
     // update hw details (lshw / lldp) - cannot proceed without this
-    val lshwData = getResource("lshw-basic.xml")
+    val lshwData = getResource(lshwXml)
     val lldpData = getResource("lldpctl-two-nic.xml")
```

Comment:
ha, for sure. I can get a second chance at travis not snuffing me on the solr test as well :laughing: